cmd/installer/pkg/install/install.go

```diff
@@ -73,11 +74,13 @@ func Install(p runtime.Platform, seq runtime.Sequence, opts *Options) (err error
 // Installer represents the installer logic. It serves as the entrypoint to all
 // installation methods.
 type Installer struct {
-	cmdline  *procfs.Cmdline
-	options  *Options
-	manifest *Manifest
-	Current  string
-	Next     string
+	cmdline    *procfs.Cmdline
+	options    *Options
+	manifest   *Manifest
+	bootloader bootloader.Bootloader
```

Comment:
Only added this. We now use an interface instead of an explicit bootloader (still a WIP).